### ISSUE TITLE
Search between_* throws away information

### ISSUE THREAD

**Issue by Shaakmiller** [facetoe/zenpy#168]
My local time-zone is UTC-5 and everything in Zendesk is UTC. If I want to ask for everything up to now, I would think I can use:

`today = datetime.datetime.now() + datetime.timedelta(hours=5) 

# OR I COULD USE:

from pytz import timezone
import pytz

utc = pytz.utc
eastern = timezone('US/Eastern')

today = eastern.localize(datetime.datetime.now())

#However it only works if I use:

today = datetime.datetime.now() + datetime.timedelta(hours=10) 

#Why do I have to use 10 hours if I am only utc-5:00?`

here is the call i am using:

`t = zenpy_client.search("",created_between=[thirty_days_withtime, today], type="ticket",group_id=[21001940, 20896804])`

Can you shed any light on this?

So greatly appreciated

**Comment by facetoe** [comment]
Thanks for the report @Shezmula, this looks like a bug in the `between_*` handling where it throws away the hour/minute information and only sends the date. If you enable logging you can see the request generated for the following code:

```python
five_days_ago, now = datetime.datetime.utcnow() - datetime.timedelta(days=5), datetime.datetime.utcnow()
for result in zenpy_client.search(created_between=(five_days_ago, now)):
    print(result)
```

is:

`https://zenpydevhelp.zendesk.com/api/v2/search.json?query=created>2017-11-09 created<2017-11-14`

According to the docs I found here (https://support.zendesk.com/hc/en-us/articles/203663226-Zendesk-Support-search-reference) it should be formatted as an `ISO8601` string, which would retain the hours etc. I'm guessing that the extra 5 hours are tipping you over the day boundary or something like that.

I'll push a fix for this when I get the chance.

**Comment by Shezmula** [comment]
Hmmmm, ok, I will see if I can write up a quick fix in the mean time. Thanks for looking into that so quickly.

**Comment by facetoe** [comment]
Fixed in #169.
